crates/infra/cli/src/toolchains/wasm/mod.rs

```diff
@@ -30,14 +31,21 @@ impl WasmPackage {
     fn generate_component(self) -> Result<PathBuf> {
         let wasm_crate = self.wasm_crate();
 
-        Command::new("cargo")
+        let mut command = Command::new("cargo")
             .arg("build")
             .property("--target", WASM_TARGET)
             .property("--package", wasm_crate)
             .flag("--all-features")
-            .flag("--release")
-            .add_build_rustflags()
-            .run();
+            .add_build_rustflags();
+
+        // add_build_rustflags() will add the release flag when running in CI, but we want the release flag to
```

Comment:
I don't think we need to add `--release` here for building locally. This will improve stack traces for panics during local development.
Not blocking of course, since this was the default behaviour before this PR.